.github/workflows/pull_request.yml

```diff
@@ -24,9 +24,17 @@ jobs:
             pull_number: context.payload.number,
             per_page: 100
           })
-          const shouldRun = result.data.filter(f => !f.filename.endsWith(".md") || !f.filename.endsWith(".txt") || !f.filename.startsWith("contrib/") || !f.filename.endsWith(".yml")).length > 0
-          console.log(shouldRun)
-          return shouldRun
+          const files = result.data.filter(file =>
+            // We wanna run this if the PR workflow is modified
+            (file.filename.endsWith("yml") && file.filename.endsWith("pull_request.yml")) ||
```

Comment:
This condition does not seem to match the comment, maybe negate the second operand? Also `".yml"` may be more precise.

Reply:
You're right, I copied in the wrong version of this code from my test repo. Fixed.